res/WELCOME.md

```diff
@@ -2,7 +2,9 @@
 
 Hi @${username}, and welcome to GitGitGadget, the GitHub App to send patch series to the Git mailing list from GitHub Pull Requests.
 
-Please make sure that your Pull Request has a good description, as it will be used as cover letter.
+Please make sure that your Pull Request has a good description, as it will be used as cover letter. You can CC potential reviewers by adding a footer to the PR description with the following syntax:
+
+    CC: Revi Ewer <revi.ewer@example.com>, Ill Takalook <ill.takalook@example.net>
```

Comment:
I love the names ;-)

Reply:
Can't take any credit there, I just copied them from https://gitgitgadget.github.io/ 😀

Reply:
🤦!